multus/multus.go

```diff
@@ -204,6 +205,34 @@ func delPlugins(exec invoke.Exec, argIfname string, delegates []*types.DelegateN
 	return nil
 }
 
+// Sits in a wait loop until a file indicating the readiness of the "default network"
+// is present (or until a user-defined timeout is reached)
+func waitForDefaultNetwork(indicatorFile string, waitSeconds int) error {
+	// If there's no file to wait for, then, this is essentially disabled.
+	if len(indicatorFile) > 0 {
```

Comment:
We can use https://godoc.org/k8s.io/apimachinery/pkg/util/wait backoff here

Reply:
Sweet! Kural thank you for the suggestion, and the awesome code example -- really appreciated.